doc/perception_pipeline/src/cylinder_segment.cpp

```diff
@@ -194,18 +219,28 @@ void cloud_cb(const sensor_msgs::PointCloud2ConstPtr& input)
     static bool points_not_found = true;
     if (points_not_found)
     {
+      // BEGIN_TUTORIAL
+      // CALL_SUB_TUTORIAL extract_cylinder
+      // Hold the relavent parameter of cylinder needed for creating a collision object
+      // There are a total of 7 parameters:
+      // 0 - radius of cylinder.
+      // 1-3 - direction vector of z-axis.
+      // 4-6 - centre point of the cylinder.
+      // 7 - height of the cylinder.
       std::vector<double> cylinder_params;
       // radius of cylinder
       cylinder_params.push_back(coefficients_cylinder->values[6]);
       // i,j,k values of the direction vector of z-axis
       cylinder_params.push_back(coefficients_cylinder->values[3]);
```

Comment:
I don't want to include these lines in the tutorial, is there a way to do that?

Reply:
The best way to do this would be to use BEGIN_SUB_TUTORIAL and END_SUB_TUTORIAL to include things you want and exclude things you don't. Check out the Python MoveGroup tutorial for an aggressive example of this method.